examples/dapperish.go

```diff
@@ -19,10 +18,10 @@ import (
 func client() {
 	reader := bufio.NewReader(os.Stdin)
 	for {
-		span := opentracing.StartTrace("getInput")
-		ctx := opentracing.BackgroundGoContextWithSpan(span)
+		ctx, span := opentracing.BackgroundContextWithSpan(
+			opentracing.StartTrace("getInput"))
 		// Make sure that global trace tag propagation works.
-		span.TraceContext().SetTraceAttribute("User", os.Getenv("USER"))
+		span.SetTraceAttribute("User", os.Getenv("USER"))
```

Comment:
Yes.
Or 

    Span.setCascadedTag("User", os.Getenv("USER"))

It's an attribute that child tags will also inherit, regardless of network/process boundaries.

I find the whole "Distributed Context Propagation" at the API level, has grown into an unneeded (psychological) [schema](https://en.wikipedia.org/wiki/Schema_%28psychology%29).

To the user of the API it can be simplified just as an attribute the casades, even over the network/process boundaries.

( The "Distributed Context Propagation" schema is still valuable in Specification though. )